programs/drift_vaults/src/state/vault_depositor.rs

```diff
@@ -16,6 +16,12 @@ use drift::math::insurance::{if_shares_to_vault_amount, vault_amount_to_if_share
 use drift::math::casting::Cast;
 use drift::math::safe_math::SafeMath;
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum WithdrawUnit {
```

Comment:
would probably put this under the VaultDepositor struct so VaultDepositor is first thing in file